App Locators/AndroidSiplyLocators.robot

```diff
@@ -0,0 +1,42 @@
+*** Settings ***
```

Comment:
Create folder for locators as well like we have created under Test

Reply:
Ok Sure

Reply:
Folder was created according to required criteria